UI/window-basic-main.cpp

```diff
@@ -7868,18 +7869,59 @@ void OBSBasic::on_actionCopyFilters_triggered()
 	ui->actionPasteFilters->setEnabled(true);
 }
 
+bool OBSBasic::QueryPasteFilters(size_t count)
+{
+	QString text;
+	if (count == 1) {
+		text = QTStr("ConfirmPasteFilters.Text");
```

Comment:
I think that prompting when pasting a filter to a single scene item is a bit disruptive.  It's not a destructive action, and it would be a workflow change.  I think the suggestion to prompt on pasting to multiple scene items was simply to prevent inadvertently pasting to a large number of scene items and having to undo them all by hand.  This may be mitigated by the upcoming undo/redo system.

Reply:
Should I just remove this case (display no prompt)?

Reply:
Yep, Just make the case `if count > 4`, show question

Reply:
Done:
https://github.com/obsproject/obs-studio/pull/4334/files#diff-4c0a52244f264feca9c3031c90edcbbbd1087a3040c15f9df4ca9700c5154fa3R7893